llama_hub/confluence/README.md

```diff
@@ -23,7 +37,7 @@ Here's an example usage of the ConfluenceReader.
 ```python
 from llama_index import download_loader
 
-ConfluenceReader = download_loader('ConfluenceReader')
+ConfluenceReader = download_loader('ConfluenceReader', refresh_cache=True)
```

Comment:
since you're in the process of landing this change, you could also try updating our default usage to 

`from llama_hub.confluence.base import ConfluenceReader` :)